Cargo.toml

```diff
@@ -8,6 +8,3 @@ members = [
     "dynamodb_lock",
 ]
 exclude = ["proofs", "delta-inspect"]
-
-[profile.dev]
-split-debuginfo = "unpacked"
```

Comment:
Not entirely sure why why had this option. however it seems it is only stable on macOS, and was causing build issues for Windows after 1,65 was released, so I removed it.